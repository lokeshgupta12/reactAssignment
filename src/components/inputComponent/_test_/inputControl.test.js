import React from 'react';
import renderer from 'react-test-renderer';
import Enzyme, { shallow } from 'enzyme';
import InputControl from '../index';
import Adapter from 'enzyme-adapter-react-16';

// Configure Enzyme adapter for Jest
Enzyme.configure({ adapter: new Adapter() });

describe('<InputControl /> component', () => {

  it('should match the snapshot', () => {
    const tree = renderer.create(<InputControl />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should render the HTML Correctly', () => {
    const wrapper = shallow(<InputControl />);

    expect(wrapper.find('input').length).toBe(1);
  });
})