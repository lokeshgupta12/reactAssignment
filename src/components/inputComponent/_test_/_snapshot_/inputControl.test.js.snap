// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InputControl /> component should match the snapshot 1`] = `
  <label />
  <input
    className="form-control"
    onChange={[Function]}
  />
`;